src/core/utils/misc.cpp

```diff
@@ -383,6 +383,16 @@ bool Utils::Misc::isPreviewable(const QString& extension)
         multimedia_extensions.insert("WAV");
         multimedia_extensions.insert("WMA");
         multimedia_extensions.insert("WMV");
+	multimedia_extensions.insert("PDF");
```

Comment:
Fix Coding Style. We don't use tab for indentation. Check out CODING_GUIDELINES.md first.